sentry/src/main/java/io/sentry/profilemeasurements/ProfileMeasurementValue.java

```diff
@@ -0,0 +1,120 @@
+package io.sentry.profilemeasurements;
+
+import io.sentry.ILogger;
+import io.sentry.JsonDeserializer;
+import io.sentry.JsonObjectReader;
+import io.sentry.JsonObjectWriter;
+import io.sentry.JsonSerializable;
+import io.sentry.JsonUnknown;
+import io.sentry.vendor.gson.stream.JsonToken;
+import java.io.IOException;
+import java.util.Map;
+import java.util.Objects;
+import java.util.concurrent.ConcurrentHashMap;
+import org.jetbrains.annotations.ApiStatus;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+
+@ApiStatus.Internal
+public final class ProfileMeasurementValue implements JsonUnknown, JsonSerializable {
```

Comment:
the docs about it is here: https://www.notion.so/sentry/Metrics-Support-91af391a3de64f97ac0d332c596aa893
The specs changed since i started the pr, but in the end we got to this point. 
We have a timestamp and a value in each `ProfileMeasurementValue`, while `MeasurementValue` contains a value and a unit.
Also, it is possible we are going to add some other fields in the future

```
struct Measurement {
    unit: String,
    values: Vec<MeasurementValue>,
}

struct MeasurementValue {
    // nanoseconds elapsed since the start of the profile
    elapsed_since_start_ns: u64,
    value: f64,
}

struct Profile {
  ...
  ...
  ...
  measurements: Option<HashMap<String, Measurement>>,
}
```